lib/signers/presign.js

```diff
@@ -51,6 +51,8 @@ function signedUrlSigner(request) {
   AWS.util.each(request.httpRequest.headers, function (key, value) {
     if (key === expiresHeader) key = 'Expires';
     if (key.indexOf('x-amz-meta-') === 0) {
```

Comment:
`if (key.toLowerCase().indexOf('x-amz-meta-') === 0)` in case the `'x-amz-meta-'` part isn't lowercased?